translator/totomlconfig/sampleConfig/complete_darwin_config.json

```diff
@@ -4,6 +4,7 @@
     "logfile": "/opt/aws/amazon-cloudwatch-agent/logs/amazon-cloudwatch-agent.log",
     "internal": true,
     "debug": true,
+    "aws_sdk_log_level": "LogDebug",
```

Comment:
I did not need to change any of the JSON sample configs.
I chose to modify the existing `translator/toenvconfig/toEnvConfig_test.go :: TestCompleteConfig()` to expect this new param, so I had to change the few JSON configs that the test uses.